Makefile

```diff
@@ -7,6 +7,8 @@ DOCKER_SQUASH ?= --squash
 VENV_DIR ?= .venv
 PIP_CMD ?= pip
 TEST_PATH ?= .
+LOCALSTACK_BUILD_GIT_HASH ?= $(shell git rev-parse --short HEAD)
```

Comment:
nitpick: Would be better to move this into the `docker-build` target, for better performance (otherwise these two commands get executed for every `make` call.